packages/dashboard/.gitignore

```diff
@@ -34,3 +34,4 @@ yarn-error.log*
 .vercel
 
 .env
+.env.ropsten
```

Comment:
instead of doing this, all env should be ignored, and only examples should be allowed... 

See https://stackoverflow.com/questions/3001888/how-do-gitignore-exclusion-rules-actually-work

Reply:
Updated PR!